app/views/org_admin/sections/_index.html.erb

```diff
@@ -0,0 +1,62 @@
+<!-- Accordion for sections -->
+<div class="row">
+  <div class="col-md-12">
+    <div class="panel-group" id="sections_accordion" role="tablist">
+      <% sections.each do |section| %>
+        <div class="panel panel-default">
+          <div class="heading-button" role="button" data-toggle="collapse"
+               data-parent="sections-accordion" 
+               href="#collapseSection<%= section.id%>"
+               aria-expanded="false" 
+               aria-controls="#collapseSection<%= section.id%>">
+
+            <div class="panel-heading" role="tab" id="<%= "headingSection#{section.id}" %>">
+              <div class="panel-title pull-left">
+                <%= section.title %>
+              </div>
+              <div class="pull-right">
+                <i class="fa fa-plus pull-right" aria-hidden="true"></i>
+              </div>
+              <div class="clearfix"></div>
+            </div>
+          </div>
+          <div
+            id="<%= "collapseSection#{section.id}" %>"
+            class="panel-collapse collapse<%= " in" if section.id == current_section %>"
+            role="tabpanel"
+            aria-labelledby="<%= "headingSection#{section.id}" %>">
+             <div class="panel-body">
+              <% if edit && section.modifiable %>
+                <%= render partial: 'org_admin/sections/edit', 
+                      locals: { 
+                        template: template, 
+                        phase: phase, 
+                        section: section, 
+                        edit: true,
+                        current_tab: current_tab } %>
+              <% end %>
+             </div>
+          </div>
+        </div>
+      <% end %>
+    </div>
+    <% if edit || template.customization_of.present? %>
```

Comment:
seeing this reminds me that we should add some sort of method in the template such as:

```
template.customised?
```